lineman/app/app.coffee

```diff
@@ -15,6 +15,9 @@ angular.module('loomioApp', ['ngNewRouter',
   renderer = new marked.Renderer()
   renderer.link = (href, title, text) ->
     "<a href='" + href + "' title='" + (title || text) + "' target='_blank'>" + text + "</a>";
+  renderer.paragraph = (text) ->
+    text = text.replace(/([\s(\[]+|^)@([a-zA-Z0-9]+)([\s)\]\,\?]|$)/g, "$1<a class='lmo-user-mention' href='/u/$2'>@$2</a>$3")
```

Comment:
Can't help but wonder if this is the reason we don't have the auto-linking functionality in beta?